apps/gitness/src/pages/run-pipeline-dialog/run-pipeline-form.tsx

```diff
@@ -112,6 +114,16 @@ export default function RunPipelineForm({
         // TODO: error toast here ?
       })
   }
+
+  const formRef = useRef<HTMLDivElement | null>(null)
+  // NOTE: custom focus implementation
```

Comment:
why do we need a custom focus implementation?
can we just set `autofocus` attribute on the first form input instead?

Reply:
Fixed. We are using setFocus that is exposed via useForm in the form library. Added autoFocusPath to RootForm.